code/builders/builder-vite/src/vite-config.ts

```diff
@@ -65,7 +65,7 @@ export async function commonConfig(
     base: './',
     plugins: await pluginConfig(options),
     resolve: {
-      conditions: ['storybook', 'stories', 'test'],
+      conditions: ['storybook', 'stories', 'test', 'browser', 'import', 'module', 'default'],
```

Comment:
As far as I understand the docs `resolve.conditions` is an **addition** instead of a replacement. Maybe you have a strong reason to apply a different resolution order then it is defined per default (import, module, browser, default). If so, can you elaborate in a comment, why this is necessary?

Reply:
This is not necessary.

Reply:
I'm in favor of keeping the code as-is because:

1. The underlaying defaults might change, without us knowing about it.

2. If there's ever a problem with the wrong file getting loaded, having this list explicit here, makes it slightly easier to understand & debug. Swapping and removing existing string in a list, is easier than understanding there's a default list added and that has an order to it too.

3. I DID have problems in the past, which prompted me to add the extra entries in the past.. 
I do not know what exact scenario that was, causing those default NOT to be included, but it seems like it's somehow possible for that not to happen.